CONTRIBUTING.md

```diff
@@ -46,76 +46,69 @@ Hi! We're really excited that you are interested in contributing to ACPaaS UI. B
     - Provide a detailed description of the bug in the PR or an accompanying [bug report issue](https://github.com/digipolisantwerp/acpaas-ui_angular/issues/new?template=bug_report.md) linked from the PR (recommended).
     - Add appropriate test coverage if applicable.
 
-- For additional guidance while developing, check the [dev guide](./guidelines/DEV_GUIDE.md).
-
 ## Development Setup
 
-You will need [Node.js](http://nodejs.org) **version 8+**.
+You will need [Node.js](http://nodejs.org) **version 10+**.
 
 After cloning the repo, run:
 
-```bash
-# install dependencies needed for the bootstrap command
-$ npm install # or yarn
-# install nested package.json dependencies
-$ npm run bootstrap
-# build into dist folder
-$ npm run build
-```
-
-Additionally, to run the examples locally:
-
-```bash
-# update the styleguide from the individual components
-$ npm run examples
-# run the styleguide examples application
-$ npm run styleguide
+```shell script
+npm install
+npm start
 ```
 
 ## Using Docker
 
 Additionally to running this project locally, the repository also offers you the option to develop using Docker containers.
 To start your service, execute:
 
-```sh
+```shell script
```

Comment:
```suggestion
```shell
```
Bron: https://github.com/highlightjs/highlight.js/blob/master/SUPPORTED_LANGUAGES.md

Reply:
Beiden zijn even juist dacht ik? Dit is ook maar automatisch door mijn editor gegenereerd